crates/libtiny_tui/src/key_map.rs

```diff
@@ -31,6 +31,7 @@ pub(crate) enum KeyAction {
     MessagesScrollBottom,
 
     Input(char),
+    RunIrcCommand(String),
```

Comment:
As mentioned in my other comment, the name is confusing. I suggest we just call this "Command".